rdt-open-reader/src/test/java/io/ona/rdt/robolectric/activity/CustomRDTCaptureActivityTest.java

```diff
@@ -68,6 +69,8 @@ public void setUp() {
     public void testOnCreateShouldCorrectlyInitializeActivity() {
         Utils.showProgressDialog(R.string.please_wait_title, R.string.please_wait_message, customRDTCaptureActivity);
         controller.create();
+        RDTApplication.getInstance().getSharedPreferences().saveLanguagePreference(BuildConfig.LOCALE);
+        io.ona.rdt.util.Utils.updateLocale(customRDTCaptureActivity);
         assertEquals(new Locale(BuildConfig.LOCALE).getLanguage(), customRDTCaptureActivity
```

Comment:
Do we need to do this? Would saving language preference before launching the activity suffice?

Reply:
You can save this preference as part of `ActivityRobolectricTest` set up.

Reply:
> Do we need to do this? Would saving language preference before launching the activity suffice?

yes we need to save this before the assertion because by default `en` will be returned by `SharedPreference` that's why we need to do this

Reply:
> You can save this preference as part of `ActivityRobolectricTest` set up.

yes it could be

Reply:
You can add a `@Before` method that all the child classes invoke.